code/game/machinery/ecto_sniffer.dm

```diff
@@ -13,10 +13,16 @@
 	var/sensor_enabled = TRUE
 	///List of ckeys containing players who have recently activated the device, players on this list are prohibited from activating the device untill their residue decays.
 	var/list/ectoplasmic_residues = list()
+	var/obj/item/radio/radio
 
 /obj/machinery/ecto_sniffer/Initialize()
 	. = ..()
 	wires = new/datum/wires/ecto_sniffer(src)
+	radio = new(src)
+	radio.keyslot = new /obj/item/encryptionkey/headset_sci
+	radio.subspace_transmission = TRUE
+	radio.canhear_range = 9
```

Comment:
This is an elegance thing, but I _think_ this will result in people with science headsets nearby hearing the message twice. 

 The sniffer already has an audio cue for those nearby and plays an animation, so this could probably be set to 0 and it'd be fine.

Reply:
Ah, I was confused by this, so thanks!